Services/UICore/classes/class.ilTemplate.php

```diff
@@ -1521,6 +1521,18 @@ function getTemplatePath($a_tplname, $a_in_module = false, $a_plugin = false)
 				$fname = "./".$module_path."templates/default/".basename($a_tplname);
 			}
 		}
+		else if(strpos($a_tplname,"src/UI")===0)
```

Comment:
Thanks for the explanation. I mean in the code. I think it would be helpful for future developers that are confronted with this code.